src/bond-sdk-cards.js

```diff
@@ -105,6 +105,103 @@ class BondCards {
     });
   }
 
+  /**
+   * The FieldParams has all parameters regarding to card field
+   * @typedef {Object} FieldParams
+   * @property {String} [htmlWrapper="text"] The expected type of response data.
+   * 'image' is wrapped in an <img src='<revealed_data>'/> HTML tag. 'text'
+   * would be inserted into a <span> element inside the iframe.
+   * @property {String} htmlSelector A selector for the field/element where the
+   * iFrame will be placed.
+   * @property {Object} [format] An objects containing a regex pattern to find and
+   * replace.
+   * @property {String} format.replaceThis String is to be replaced with the
+   * new value. Please use the format where regexp is not enclosed between
+   * slashes but do use quotation marks. ex: '(\\d{4})(\\d{4})(\\d{4})(\\d{4})'
+   * @property {String} format.withThis The string that replaces the substring
+   * specified by the specified regexp. ex: '$1-$2-$3-$4'
+   * @property {String} [format.count] Optional, defines how many times a certain
+   * string should be replaced.
+   * @property {Object} [css={}] An object of CSS rules to apply to the response.
+   */
+
+  /**
+   * The FieldType
+   * @typedef {('number'|'cvv'|'expiry')} FieldType
```

Comment:
I like using this fieldType. Can you update the jsDoc code for .show to match this?

Reply:
Sure